### DISCUSSION TITLE
Display the image inside the form

### DISCUSSION THREAD

**Discussion by sh4msi** [filamentphp/filament#4312] [Help] [answered]
Hi,
How can I display a image inside the form (I just want to read from a URL and display it)
(something like ImageColumn)

Of course, I know that it can be displayed with a custom field, but I want to know if there is a predefined solution?
And if it doesn't exist, isn't it better to add it?

**Comment by cobraprojects** [comment, marked as answer]
Using `Placeholder` layout

```
Placeholder::make('image')
           ->content(
               fn (Model $record) => new HtmlString('<img src="' . $record->image . '" alt="" />');
           )
```

if you don't need the record data you can return `HtmlString` directly without a closure.

https://filamentphp.com/docs/2.x/forms/layout#placeholder
